uqcsbot/scripts/trivia.py

```diff
@@ -156,7 +171,10 @@ def handle_question(channel: Channel, args: argparse.Namespace):
 
     # Get the answer message
     if question_data.is_boolean:
-        answer_text = f':{BOOLEAN_REACTS[0]}:' if question_data.correct_answer == 'True' else f':{BOOLEAN_REACTS[1]}:'
```

Comment:
Why do you hate ternary operators so much?

Reply:
`uqcsbot/scripts/trivia.py:159:101: E501 line too long (118 > 100 characters)`

Reply:
Also, SQL's `between` is a ternary operator; FORTH's `*/` is a ternary operator.